src/main/webapp/script.js

```diff
@@ -5,6 +5,13 @@ const loadingId = "loadingIcon";
 const channelId = "channel";
 const noCodesNode = document.createTextNode("Sorry! We didn\'t find any codes.");
 
+function checkEnterKeytoSearch() {
+    // 13 is the "Enter" key on the keyboard.
+    if (event.keyCode === 13) {
```

Comment:
instead of commenting, extra it into a constant like ENTER_KEY_CODE. https://google.github.io/styleguide/jsguide.html#naming-constant-names

Reply:
okay so previously you had to actually click the "submit" key.  can you add a comment as to why this is necessary (say it's an alternate way to submit the form)?  

optional: Instead of calling displayCodes is there a way to trigger a click on the submit button?   (That way any actions you add to the submit button get called, and if you were writing any tests verifying that the submit button gets clicked.)

Reply:
Done and Done.